src/Whoops/Util/TemplateHelper.php

```diff
@@ -59,6 +63,42 @@ public function escapeButPreserveUris($raw)
     }
 
     /**
+     * Format the given value into a human readable string.
+     *
+     * @param  mixed $value
+     * @return string
+     */
+    public function dump($value)
+    {
+        if (class_exists('Symfony\Component\VarDumper\Cloner\VarCloner')) {
+            $cloner = new VarCloner();
+            if ('cli' === PHP_SAPI) {
```

Comment:
To be honest, I copied this part out of `VarDumper::dump` without further thinking.. Atm we only use this viewHelper in web context, so actually we can drop the cli-sapi case.